.travis.yml

```diff
@@ -1,13 +1,10 @@
 language: ruby
 
 rvm:
-  - 2.0
   - 2.1
-  - 2.2
+  - 2.2.3
+  - 2.3.0
 gemfile:
-  - gemfiles/Gemfile.ar-3.2
-  - gemfiles/Gemfile.ar-4.0
-  - gemfiles/Gemfile.ar-4.1
   - gemfiles/Gemfile.ar-4.2
```

Comment:
The `ActiveRecord::Type.lookup` API that I recommended would not be available in Rails 4.2. I also feel that we are not relying in the type API as much we can, but that is just a feeling. If we can use `ActiveRecord::Type.lookup` we can probably use conditional code too, so not a big deal.